frontend/src/metabase/containers/QuestionLoader.jsx

```diff
@@ -71,7 +71,7 @@ const QuestionLoader = ({
       children={children}
     />
   ) : // if there's a questionHash it means we're in ad-hoc land
-  questionHash != null ? (
+  questionHash != null && questionHash !== "" ? (
```

Comment:
What's the reason for the additional check?

Reply:
This was a code from the previous PR. Paul explained it here:
https://github.com/metabase/metabase/pull/14224#discussion_r549896224

Reply:
Ah got it. Thanks.